morf-h2/src/test/java/org/alfasoftware/morf/jdbc/h2/TestH2Dialect.java

```diff
@@ -31,7 +31,7 @@
  *
  * @author Copyright (c) Alfa Financial Software 2010
  */
-public class TestH2Dialect extends AbstractSqlDialectTest {
+public class TestH2Dialect extends AbstractSqlDialectTestWithNoSupportForWindowFunctions {

```

Comment:
I thought H2 did support WindowFunctions?

Reply:
Actually speaking all the ones that are showing as unsupported for window functions now support Window functions. 
H2 from 1.4.198 onwards (currently using 1.4.200)
MYSQL : support is provided in 8.0 which is the current version
SqlServer : All supported versions.

Which means I can remove the unsupported exception in all these dialects. It also means that I would not need to do all the hacks to get the tests working.